tests/widget/modal/widget-modal.login-social-redirect.test.js

```diff
@@ -6,7 +6,7 @@ test('Modal widget with social callback redirect', async ({ page }) => {
   const loginButton = page.getByRole('button', { name: 'Open Login Modal' });
   await loginButton.click();
 
-  const button = await page.locator('button', { hasText: 'Sign in with Google' });
+  const button = await page.locator('button', { hasText: 'Continue with Google' });
```

Comment:
when we pass over code like this, i think its a good idea to update the api to the newer style.

this should probably be `page.getByRole`

Reply:
Good point. It's weird, I thought I fixed references to this in the previous PR. Anyway I'll fix it now. Thanks.